trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require './decorator'
+
+class TrimmerDecorator < Decorator
+  def correct_name
+    @nameable.correct_name.slice(0, 10)
```

Comment:
[REQUIRED] Kindly fix this code because the logic is wrong. It would start slicing if the length of the word is eleven. Don't forget that Ruby is a zero-indexed language.